Source/player.cpp

```diff
@@ -905,8 +905,8 @@ void __fastcall InitPlayer(int pnum, BOOL FirstTime)
 	plr[pnum]._pInvincible = FALSE;
 
 	if ( pnum == myplr ) {
-		deathdelay = FALSE;
-		deathflag = 0;
+		deathdelay = 0;
+		deathflag = FALSE;

```

Comment:
I haven't checked further, but I think @squidcc thought `deathflag` is an integer rather than a boolean.

Reply:
deathflag is fine (as BOOL), deathdelay is the int (it gets assigned 30 at one point).

Reply:
Ah, ok.